text/0308-cli-advisories.md

```diff
@@ -0,0 +1,263 @@
+# CLI advisories
+
+* **Original Author(s)**: [@otaviomacedo](https://github.com/otaviomacedo)
+* **Tracking Issue**: [#308](https://github.com/aws/aws-cdk-rfcs/issues/308)
+* **API Bar Raiser**: @{BAR_RAISER_USER}
+
+A new CLI feature to notify customers about urgent and important issues that
+require their attention.
+
+## Working backwards
```

Comment:
This is the "working backwards" from a user perspective (the README).

I would add another working backwards section which describes the publisher's experience (e.g. the content we will have in our runbook for how to publish an advisory).

Reply:
Done.